CK3toEU4/Source/Configuration/Configuration.h

```diff
@@ -98,6 +98,9 @@ class Configuration: commonItems::parser
 	void setOutputName();
 	void verifyCK3Path();
 	void verifyEU4Path() const;
+	void verifyCK3Version(const mappers::ConverterVersion& converterVersion) const;
+	void verifyEU4Version(const mappers::ConverterVersion& converterVersion) const;
+	[[nodiscard]] std::optional<GameVersion> getRawVersion(const std::string& filePath) const;
```

Comment:
Put above verifyCK3Version and verifyEU4Version to match the order in the .cpp file.